classes/Repository/ModuleRepository.php

```diff
@@ -73,4 +87,40 @@ public function getInformationsAboutModule(): array
             'upgradeLink' => $this->getUpgradeLink(),
         ];
     }
+
+    /**
+     * @return array
+     */
+    public function getModuleDebug(): array
+    {
+        return [
+            'isEnabled' => $this->moduleIsEnabled(),
+            'hooks' => $this->getActiveHooks(),
+            'enableLink' => $this->getActiveLink(),
+        ];
+    }
```

Comment:
At the first glance, the name of this method caught my eye. Having the debug word in it although it is not used eslewhere than in the debug tab is confusing. It should be "getModuleStatus" or something like that, and we already have a similar method right about this method (getInformationsAboutModule)

So far it seems `'hooks' => $this->getActiveHooks(),` could be moved in the existing call of debug data, and the rest could be moved in all the JS data we get when we refresh the page. No need of a new getter, action and so on.